src/main/java/frc/robot/commands/DistanceDriveCommand.java

```diff
@@ -5,47 +5,60 @@
 package frc.robot.commands;
 import frc.robot.subsystems.Drivetrain;
 import frc.robot.Constants.AutoConstants;
-
+import frc.robot.Constants.DrivetrainConstants;
 
 //import java.lang.Math.*;
 
 import edu.wpi.first.wpilibj2.command.Command;
 
 
-/** An taxi command that uses the drivetrain subsystem. */
+/** A dead reckoning taxi command that uses the drivetrain subsystem. */
 public class DistanceDriveCommand extends Command {
   @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
   private final Drivetrain m_drivetrain;
-  private double m_rightEncoderStart;
-  private double m_leftEncoderStart;
+  // private double m_rightEncoderStart;
```

Comment:
why is this commented out?

Reply:
instead of getting the start positions, we're just resetting the encoders back to 0 every time we call a new command for simplicity. I can get rid of those comments now